### ISSUE TITLE
Updating to 2.0.0 removed all existing DeathChests and not spawning any new ones

### ISSUE THREAD

**Issue by heinermann** [DevCyntrix/death-chest#63]
All existing death chests on the server were deleted when I shut it down and replaced 1.5.24 with 2.0.0. In addition to this, it was not spawning any new death chests and instead deleting the items entirely. I only replaced the jar, all other files remained as they were.

The nature of the new death was fall damage over a manual sugarcane farm.

Previous server restart:

```
[19:36:29] [Server thread/INFO]: [DeathChest] Enabling DeathChest v1.5.24
[19:36:29] [Server thread/INFO]: Environment: authHost='https://authserver.mojang.com', accountsHost='https://api.mojang.com', sessionHost='https://sessionserver.mojang.com', servicesHost='https://api.minecraftservices.com', name='PROD'
[19:36:30] [Server thread/INFO]: [DeathChest] 15 death chests loaded.
```

After updating:

```
[14:21:09] [Server thread/INFO]: [DeathChest] Enabling DeathChest v2.0.0
[14:21:10] [Server thread/INFO]: [DeathChest] [STDOUT] Migrating the deathchests...
[14:21:10] [Server thread/WARN]: Nag author(s): '[CyntrixAlgorithm]' of 'DeathChest v2.0.0' about their usage of System.out/err.print. Please use your plugin's logger instead (JavaPlugin#getLogger).
[14:21:10] [Server thread/INFO]: Environment: authHost='https://authserver.mojang.com', accountsHost='https://api.mojang.com', sessionHost='https://sessionserver.mojang.com', servicesHost='https://api.minecraftservices.com', name='PROD'
[14:21:10] [Server thread/INFO]: [DeathChest] [STDOUT] 15 chests found
[14:21:10] [Server thread/INFO]: [DeathChest] [STDOUT] deserializing...
[14:21:10] [Server thread/INFO]: [DeathChest] [STDOUT] deserializing...
[14:21:10] [Server thread/INFO]: [DeathChest] [STDOUT] deserializing...
[14:21:10] [Server thread/INFO]: [DeathChest] [STDOUT] deserializing...
[14:21:10] [Server thread/INFO]: [DeathChest] [STDOUT] deserializing...
[14:21:10] [Server thread/INFO]: [DeathChest] [STDOUT] deserializing...
[14:21:10] [Server thread/INFO]: [DeathChest] [STDOUT] deserializing...
[14:21:10] [Server thread/INFO]: [DeathChest] [STDOUT] deserializing...
[14:21:10] [Server thread/INFO]: [DeathChest] [STDOUT] deserializing...
[14:21:10] [Server thread/INFO]: [DeathChest] [STDOUT] deserializing...
[14:21:10] [Server thread/INFO]: [DeathChest] [STDOUT] deserializing...
[14:21:10] [Server thread/INFO]: [DeathChest] [STDOUT] deserializing...
[14:21:10] [Server thread/INFO]: [DeathChest] [STDOUT] deserializing...
[14:21:10] [Server thread/INFO]: [DeathChest] [STDOUT] deserializing...
[14:21:10] [Server thread/INFO]: [DeathChest] [STDOUT] deserializing...
[14:21:10] [Server thread/INFO]: [DeathChest] 0 death chests loaded in world "world"
[14:21:10] [Server thread/INFO]: [DeathChest]   Of these 0 have expired
[14:21:10] [Server thread/INFO]: [DeathChest] 0 death chests loaded in world "world_nether"
[14:21:10] [Server thread/INFO]: [DeathChest]   Of these 0 have expired
[14:21:10] [Server thread/INFO]: [DeathChest] 0 death chests loaded in world "world_the_end"
[14:21:10] [Server thread/INFO]: [DeathChest]   Of these 0 have expired
```

Full log: [2023-09-05-2.log](https://github.com/DevCyntrix/death-chest/files/12526573/2023-09-05-2.log)
Config: [config.yml.txt](https://github.com/DevCyntrix/death-chest/files/12526600/config.yml.txt)

**Comment by DevCyntrix** [comment]
Hi Adam, 
Thank you for the report. I think the plugin had a problem while migrating the chests from the one format to the other. Would you like to contact me on discord?

**Comment by DevCyntrix** [comment]
Fixed in 2.0.1
